styleguide/derek/_partials/_footer.jade

```diff
@@ -1,2 +1,2 @@
-.container-fluid.bg-blue-gray.standard-padding
+.container-fluid.bg-blue-gray.fluid-footer-container
```

Comment:
@amberRrucker won't that get confusing?

I think it would be better to have partials included in the layout like `_footer` is as they are all distinct:

https://github.com/rackerlabs/zoolander/blob/697650c11fa5c433bbf4639e6b70db7e2a697a3d/styleguide/derek/examples/_layout.jade#L6